src/DynamoCore/Core/Threading/UpdateRenderPackageAsyncTask.cs

```diff
@@ -228,12 +231,13 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, ref List<str
                         var nEnd = plane.Origin.Add(plane.Normal.Scale(2.5));
                         package.PushLineStripVertex(nEnd.X, nEnd.Y, nEnd.Z);
 
-                        for (var i = 0; i < package.LineStripVertices.Count/3/2; i++)
+                        var lineVertCounts = package.LineStripVertices.Count;
+                        for (var i = 0; i < lineVertCounts / 3 / 2; i++)
                         {
                             package.PushLineStripVertexCount(2);
                         }
 
-                        for (var i = 0; i < (package.LineStripVertices.Count/3)*4; i += 4)
+                        for (var i = 0; i < (lineVertCounts / 3) * 4; i += 4)
                         {
                             package.PushLineStripVertexColor(180,180,180,255);
```

Comment:
I think I'm missing something. `lineVertCounts` can't be const. We're getting it from the number of vertices in the package which will change at runtime.